brain.js

```diff
@@ -1,19 +1,24 @@
-var Brain = function(divID, fnPlot, manifest_url) {
-	this.selectedLabel = null;
-	this.fnPlot = fnPlot;
-	this.divID = divID;
-	this.manifest_url = (manifest_url) ? manifest_url : "files_to_load.json";
-	
-	this.container_size = function() {
-		var container_pos = this.container.getBoundingClientRect();  // [top, left, right, bottom]
-		return [container_pos.width, container_pos.height];
-	};
+var Brain = function(kwargs) {
+	//divID, fnPlot, manifest_url
 
+	var _this = this;
+	this.selectedLabel = null;
+	this.fnPlot = kwargs.callback || null;
+	this.divID = kwargs.divID || 'brain';
+	this.manifest_url = kwargs.manifest || "files_to_load.json";
+	this.data_url = kwargs.data_url || 'data/';
```

Comment:
This is how default args are done: get the object prop, or default value. Succinct, readable, and works great!